tests/unit/test_rfdetr_pose.py

```
@@ -1,540 +1,236 @@
+        num_windows=2,
+        dec_layers=1,
+        num_queries=_GROUP_DETR,
+        num_select=_GROUP_DETR,
+        projector_scale=("P4",),
+        out_feature_indexes=(3, 6, 9, 12),
+        resolution=_RES,
+        positional_encoding_size=_PE,
+    )
+
+
+def _build_pose_model(num_keypoints_per_class=(0, 17), nb_classes=2):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_build_pose_model`

(ANN202)